tests/lib/config/plugins.js

```diff
@@ -88,6 +91,63 @@ describe("Plugins", function() {
             }, /Failed to load plugin/);
         });
 
+        it("should load a scoped plugin when referenced by short name", function() {
+            StubbedPlugins.load("@scope/example");
+            assert.equal(StubbedPlugins.get("@scope/example"), scopedPlugin);
+        });
+
+        it("should load a scoped plugin when referenced by long name", function() {
+            StubbedPlugins.load("@scope/eslint-plugin-example");
+            assert.equal(StubbedPlugins.get("@scope/example"), scopedPlugin);
+        });
+
+        it("should register environments when scoped plugin has environments", function() {
+            scopedPlugin.environments = {
+                foo: {}
+            };
+            StubbedPlugins.load("@scope/eslint-plugin-example");
+
+            assert.equal(Environments.get("@scope/example/foo"), scopedPlugin.environments.foo);
+        });
+
+        it("should register rules when scoped plugin has rules", function() {
+            scopedPlugin.rules = {
+                foo: {}
+            };
+            StubbedPlugins.load("@scope/eslint-plugin-example");
+
+            assert.equal(Rules.get("@scope/example/foo"), scopedPlugin.rules.foo);
+        });
+
+        describe("(NOTE: those behavior will be removed by 4.0.0)", () => {
```

Comment:
Oh, actually `it(...)`s were copied from the tests of non-scoped. I wrote only this `describe(..)` newly. Then, I forgot to check copied code!
I will fix those.